python/setup.py

```diff
@@ -577,7 +577,8 @@ def has_ext_modules(foo):
                      },
     setup_requires=['setuptools_scm', 'cython >= 0.27'] + setup_requires,
     install_requires=install_requires,
-    tests_require=['pytest', 'pandas', 'pathlib2; python_version < "3.4"'],
+    tests_require=['pytest', 'pandas', 'hypothesis',
+                   'pathlib2; python_version < "3.4"'],
```

Comment:
Possible to maintain one list of the dependencies?

Reply:
In my perspective setup.py is the ideal place to maintain dependencies, and `requirements<arbitrary-postfix>.txt` files are just freezed pip environments, similarly to `conda_env_cpp.yml` or `conda_env_python.yml`.

However We should keep reducing the `pip install <packages...>` and `conda install <packages...>` statements in the codebase.